sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c

```diff
@@ -1608,18 +1612,20 @@ static void _sched_next_frame(gnrc_sixlowpan_frag_fb_t *fbuf)
 
 static void _sched_arq_timeout(gnrc_sixlowpan_frag_fb_t *fbuf, uint32_t offset)
 {
-    if (fbuf->sfr.arq_timeout_event.msg.content.ptr != NULL) {
-        DEBUG("6lo sfr: ARQ timeout for datagram %u already scheduled\n",
-              (uint8_t)fbuf->tag);
-        return;
+    if (!IS_ACTIVE(CONFIG_GNRC_SIXLOWPAN_SFR_MOCK_ARQ_TIMER)) {
```

Comment:
```suggestion
    if (IS_ACTIVE(CONFIG_GNRC_SIXLOWPAN_SFR_MOCK_ARQ_TIMER)) {
        return;
    } 
```

Less nesting makes this easier to read